src/apps/umami/arbitrum/umami.compound.token-fetcher.ts

```diff
@@ -76,13 +103,14 @@ export class ArbitrumUmamiCompoundTokenFetcher implements PositionFetcher<AppTok
       price,
       tokens,
       dataProps: {
-        pricePerShare: `${pricePerShare}`,
+        pricePerShare: `${parseFloat(`${pricePerShare}`).toFixed(3)}`,
       },
```

Comment:
You probably don't need `pricePerShare` in the `dataProps` at all since you have it already on line 102.

Reply:
Indeed, removed and fixed the other one